README.md

```diff
@@ -23,7 +23,7 @@ Clone the repository including submodules
 
 Build and run
 
-- Using Visual Studio 2017, Rider or Visual Studio Code (configurations are included)
+- Using Visual Studio 2017, Rider or Visual Studio Code (configurations are included. No exe file will be generated, simply "dotnet restore" in osu folder then "dotnet run" in osu.desktop folder)
```

Comment:
Move this a line down. All of those IDEs do this automatically. ``dotnet restore`` is also automatically executed by ``dotnet run``

Reply:
Not 100% sure what you meant by moving a line down.
Sorry, I hope this is fine now.